war/src/main/less/base/style.less

```diff
@@ -460,74 +442,25 @@ pre.console {
 }
 
 div.behavior-loading {
-    position: absolute;
-    left: 0;
-    right: 0;
-    width: 100%;
-    height: 100%;
-    background-color: #e4e4e4;
-    text-align: center;
-    // TODO: use a display font, maybe 3rem
-    font-size: 300%;
-    opacity: 0.5;
-    -ms-filter: "progid:DXImageTransform.Microsoft.Alpha(Opacity=50)";
-    filter: alpha(opacity=50);
-}
+  position: fixed;
```

Comment:
This is a bit problematic IMO, is there no other way? It's a common source of performance issues, and I'd be worried about that, especially if it has an animated item within.

Reply:
> The spinner feels a bit fast to me, looks weird next to the build status spinning at a much slower rate anyone else feel this? seems really good otherwise
> 
>  new-spinner.mov 
> 
> FTR pipeline will need a separate PR to maintain consistency, could you file a PR there to please? Once this is merged we can look at getting it released, ideally it will be compatible across Jenkins versions in some way (maybe a version check)

I've slowed it down and also added a check for users with [prefers-reduced-motion](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion) enabled, in which case they'll get an even slower spinner